### PR TITLE
Remove support for translations-xx format

### LINKED ISSUE #1698: Extensibility: Remove language-specific translation sections
Get rid of the `translation-{lang}` sections from the extension configuration. Leave only the generic `translations`.

### PR DESCRIPTION
**Pull request by wchrapka**
**Description**

Removes the alternative version of adding the EXT translations through:
```
data:
  translations-en: {}
  translations-pl: {}
```

**Related issue(s)**

Resolves #1698

### CODE REVIEW

File: core/src/luigi-config/customResources.js [hunk 1]
```diff
@@ -58,35 +58,22 @@ async function loadTargetClusterCRs(authData) {
...
-                translations = merge(translations, {
-                  [lang]: langTranslations,
-                });
-              } else {
-                translations = merge(translations, langTranslations);
-              }
-              return { ...acc, translations };
+    .map(configMap => {
+      const cr = Object.fromEntries(
+        Object.entries(configMap?.data || []).map(
```
Comment (Wawrzyn321) [inline core/src/luigi-config/customResources.js:63]:
```suggestion
        Object.entries(configMap?.data || {}).map(
```
`configMap.data` cannot be an array.

Comment (Wawrzyn321) [reply core/src/luigi-config/customResources.js:63]: I mean, `Object.entries([]) `is the same as `Object.entries({})`...

File: docs/extensibility/resources.md [hunk 2]
```diff
@@ -254,7 +254,7 @@ Those fields are used to build the related resource URL and filter the received
 
 ## translations sections
```
Comment (Wawrzyn321) [inline docs/extensibility/resources.md:255]:
```suggestion
## translations section
```

**Review by Wawrzyn321 — DISMISSED** [review]

**Review by grego952 — APPROVED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**